apps/mobile/src/components/Notification/NotificationRegistrar.tsx

```diff
@@ -0,0 +1,29 @@
+import { useNavigation } from '@react-navigation/native';
+import * as Notifications from 'expo-notifications';
+import { useEffect } from 'react';
+import { useRelayEnvironment } from 'react-relay';
+
+import { registerNotificationToken } from '~/components/Notification/registerNotificationToken';
+import { RootStackNavigatorProp } from '~/navigation/types';
+
+export function NotificationRegistrar() {
+  const relayEnvironment = useRelayEnvironment();
+
+  useEffect(() => {
+    registerNotificationToken({ shouldPrompt: false, relayEnvironment });
+  }, [relayEnvironment]);
+
+  const navigation = useNavigation<RootStackNavigatorProp>();
+  useEffect(() => {
+    const cleanup = Notifications.addNotificationResponseReceivedListener(() => {
+      navigation.navigate('MainTabs', {
```

Comment:
navigate to the notifications screen when the user presses a notification. 

use a random fetch key so we definitely get a refresh